src/util/uint.rs

```diff
@@ -319,8 +315,8 @@ macro_rules! construct_uint {
             }
         }
 
-        impl fmt::Debug for $name {
-            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        impl ::std::fmt::Debug for $name {
```

Comment:
Got the same of consistency, can we make these also `$crate::std::...`?

Reply:
I think `::std::` may be preferable for the standard library, since not all of the downstream mods will be directly importing it. So now I have use `$crate::` only for the crate-specific functionality. 

AFAIR, Rust docs for macros use "::std" form

Probably it will be nice to change everywhere throughout the code, I can work on that.